doc/man/8/mount.ceph.rst

```diff
@@ -1,182 +1,216 @@
 :orphan:
 
 ========================================
- mount.ceph -- mount a ceph file system
+ mount.ceph -- mount a Ceph file system
 ========================================
 
 .. program:: mount.ceph
 
 Synopsis
 ========
 
-| **mount.ceph** [*monaddr1*\ ,\ *monaddr2*\ ,...]:/[*subdir*] *dir* [
+| **mount.ceph** [*mon1_socket*\ ,\ *mon2_socket*\ ,...]:/[*subdir*] *dir* [
+  -o *options* ]
+
+| **mount** -t ceph [*mon1_socket*\ ,\ *mon2_socket*\ ,...]:/[*subdir*] *dir* [
```

Comment:
It's not typical to add /bin/mount command syntax here. It's not wrong per-se, just unusual. Re: mount.nfs(8) and mount.cifs(8).

Reply:
Removed.